### ISSUE TITLE
Class Not Found Exception

### ISSUE THREAD

**Issue by Mihai7** [Stratio/Spark-MongoDB#107]
I am trying to use the connector in order to get data from mongodb inside a Java Spark Client. I have the Stratio dependecy in my pom, is there anythng else I should add in order to run a basic query?

**Exception in thread "main" java.lang.ClassNotFoundException: Failed to find data source: com.stratio.datasource.mongodb. Please find packages at http://spark-packages.org at org.apache.spark.sql.execution.datasources.ResolvedDataSource$.lookupDataSource(ResolvedDataSource.scala:77)**

**Comment by Mihai7** [comment]
Problem solved by including the --packages property when submiting the job packaged as a jar. Is there a way to set this from code?

**Comment by pmadrigal** [comment]
Hi @Mihai7 ,

Try including all the dependencies in your jar.

Hope help you!

**Comment by pfcoperez** [comment]
@Mihai7 Did @pmadrigal suggestion solve your problem?

**Comment by Mihai7** [comment]
Yes, if i create a fat jar it works. However, this takes time to compile and leads to a big jar (over 100 MB).  I feel that there should be a easier way so I could run my code from inside the IDE (inteliji).

**Comment by pfcoperez** [comment]
@Mihai7 If you don't want to use a fat jar, list each required dependency in your spark driver's pom
